modules/cluster-logging-deploy-console.adoc

```diff
@@ -229,19 +229,21 @@ The number of primary shards for the index templates is equal to the number of E
 
 .. Select the *openshift-logging* project.
 +
-You should see several pods for OpenShift Logging, Elasticsearch, Fluentd, and Kibana similar to the following list:
-+
-* cluster-logging-operator-cb795f8dc-xkckc
-* collector-pb2f8
-* elasticsearch-cdm-b3nqzchd-1-5c6797-67kfz
-* elasticsearch-cdm-b3nqzchd-2-6657f4-wtprv
-* elasticsearch-cdm-b3nqzchd-3-588c65-clg7g
-* fluentd-2c7dg
-* fluentd-9z7kk
-* fluentd-br7r2
-* fluentd-fn2sb
-* fluentd-zqgqx
-* kibana-7fb4fd4cc9-bvt4p
+You should see several pods for OpenShift Logging, Elasticsearch, the log collector, and Kibana, similar to the following list:
```

Comment:
Suggestion - I think this could be renamed as '_You should see the OpenShift Logging stack components like Elasticsearch, collector and Kibana successfully running, similar to the following list:_"